bootstrap/manifest_template.json

```diff
@@ -77,6 +77,10 @@
         "scheduler": {
           "cmd": ["scheduler"],
           "omni": true
+        },
+        "deployer": {
+          "cmd": ["deployer"],
+          "omni": true
```

Comment:
If the host that contains the deployer dies, we do want to continue to be able to deploy (same reasoning as for the scheduler being omni).